### PR TITLE
[FIX] Invalid runtime API usage when retrieving binary payload from a mime-datasource

### PR DESCRIPTION
**Pull request by ayeshLK**
Fixes [#3662](https://github.com/ballerina-platform/ballerina-standard-library/issues/3662)
Related to [#208](https://github.com/wso2-enterprise/internal-support-ballerina/issues/208)

## Examples
N/A

## Checklist
- [X] Linked to an issue
- [X] Updated the changelog
- [ ] ~Added tests~
- [ ] ~Updated the spec~
- [ ] ~Checked native-image compatibility~

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ This file contains all the notable changes done to the Ballerina MIME package th
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to 
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+- [Binary payload retrieved from the `http:Request` has different content-length than the original payload](https://github.com/ballerina-platform/ballerina-standard-library/issues/3662)
```
Comment (TharmiganK) [inline changelog.md:8]:
```suggestion
```

File: changelog.md [hunk 2]
```diff
@@ -4,6 +4,11 @@ This file contains all the notable changes done to the Ballerina MIME package th
...
+
+## Fixed
+
```
Comment (TharmiganK) [inline changelog.md:11]:
```suggestion
- [Binary payload retrieved from the `http:Request` has different content-length than the original payload](https://github.com/ballerina-platform/ballerina-standard-library/issues/3662)

```

File: changelog.md (hunk 1 above)
Comment (ayeshLK) [reply changelog.md]: Fixed with [4047c9497edf54c385fea1af1022e8c317e871a4](https://github.com/ballerina-platform/module-ballerina-mime/pull/467/commits/4047c9497edf54c385fea1af1022e8c317e871a4)

File: changelog.md (hunk 2 above)
Comment (ayeshLK) [reply changelog.md:11]: Fixed with [4047c9497edf54c385fea1af1022e8c317e871a4](https://github.com/ballerina-platform/module-ballerina-mime/pull/467/commits/4047c9497edf54c385fea1af1022e8c317e871a4)

**Review by chamil321 — APPROVED** [review]

**Pull request merged**